test/distributions/corrado/factoryMethods/typeMethods.comm-none.good

```diff
@@ -130,7 +130,7 @@ Block:
 0 0 0 0 0 0
 0 0 0 0 0 0
 {1..6, 1..6}
-[BlockDom(2,int(64),one,unmanaged DefaultDist)] int(64)
+[BlockDom(2,int(64),one,unmanaged DefaultDist)] eltType
```

Comment:
Hrm.  It seems unfortunately to lose the type information output validation here.  I see that's coming from the prediff you added, though, so it seems like an intentional choice, walk me through it?  It's interesting that none of the output changed involved `int(16)`, too.

Reply:
Basically, I did not want to duplicate the .good files for each `eltType` specified in the `.compopts`. Since the test prints out the array types, it either needs to be prediffed to always match on the `eltType`, or the .good files need to be duplicated.

Reply:
I understand that, but it could potentially prevent us from noticing an actual problem if somehow the wrong type was being used (especially in the atomics case).  Given that the `atomic` case is doing unnecessary additional work with `eltType2`, maybe that means the atomic case should be its own, smaller test file instead?

Reply:
I don't see the `atomic` case doing additional work as that bad, but I agree with your sentiment about the prediff preventing an actual problem form surfacing. 

I think duplicating the good files is probably less maintenance than having a separate test

Reply:
> I don't see the `atomic` case doing additional work as that bad

It's definitely not that bad, I'm approaching it more from the perspective of "well, if we don't need to be doing everything the original test is doing, then having separate .good files is more justifiable.  Plus it'll help distinguish what's happening with failures modes more easily to have separate, descriptive test names".  But I'm okay with just having additional .good files for the same test - at the end of the day, the coverage is the most important part

Reply:
Done in https://github.com/chapel-lang/chapel/pull/25709/commits/1aa786c07c7b6838647c755513e0b4c9ed810cac